some work on archetypes (#834)
ropensci/roweb3: content/blog/2024-08-28-blop/index.qmd

---
slug: "post-template"
title: Post Title in Title Case
# Delete the package_version line below if your post is not about a package
package_version: 0.1.0
author:
  - Maëlle Salmon
# Set the date below to the publication date of your post
date: 2024-08-28
# Minimal tags for a post about a community-contributed package 
# that has passed software peer review are listed below
# Consult the Technical Guidelines for information on choosing tags
tags:
  - Software Peer Review
  - packages
  - R
  - community
# The summary below will be used by e.g. Mastodon preview cards
description: "A very short summary of your post (~ 100 characters)"
# If you have no preferred image for Mastodon preview cards,
# delete the socialImg and socialAlt lines below 
# - Note "/" between year/month/day
socialImg: blog/2019/06/04/post-template/name-of-image.png
socialAlt: "Alternative description of the image"
# the text below is for populating the "share on Twitter" button
# if deleted, the title of the post will be used
social: "A post about blabla by @username@server.extension!"
format: hugo-md
---

> This is the R Markdown (.Rmd) template for a blog post or tech note. 

Reviewer: ```suggestion
This is the Quarto (.qmd) template for a blog post or tech note. 
```

To generate your post with just Markdown (.md), use that template instead.

Throughout this template, including the YAML, 
you should change "post-template" to the slug of your post, 
and "2019-06-04" to your publication date.

> Save this file under /content/blog/YYYY-MM-DD-slug/index.Rmd in the local copy of your roweb3 fork.

Reviewer: ```suggestion
Save this file under /content/blog/YYYY-MM-DD-slug/index.qmd in the local copy of your roweb3 fork.
```

> The chunk below ensures your plots will be inserted in the same folder as your Rmd so you can submit the bundle (.Rmd, .md, images) to roweb3.

Reviewer: ```suggestion
The chunk below ensures your plots will be inserted in the same folder as your qmd so you can submit the bundle (.qmd, .md, images) to roweb3.
```
